android/src/main/java/com/magicleap/magicscript/scene/nodes/UiImageNode.kt

```diff
@@ -136,4 +140,12 @@ open class UiImageNode(
         }
     }
 
+    private fun setIsOpaque(props: Bundle) {
+        props.read<Boolean>(PROP_OPAQUE)?.let { isOpaque ->
+            if (isOpaque) {
```

Comment:
This will be set only once? If there is redrawing, I think we should remove background if is not opaque

Reply:
Good catch, I missed it